lib/server-code/runners/cloud-workers-broker.js

```diff
@@ -75,7 +101,8 @@ class WorkersBroker extends EventEmitter {
         if (worker.heartbeat + this.heartbeatTimeout < Date.now()) {
           this.killWorker(worker, (
             `Worker expired due to heartbeat timeout (${this.heartbeatTimeout}ms), ` +
-            `the last ping was: ${worker.heartbeat && new Date(worker.heartbeat).toISOString()}`
+            `the last ping was: ${worker.heartbeat && new Date(worker.heartbeat).toISOString()} ` +
```

Comment:
also would be nice to improve the date part. 
let's show `time ago` instead of full date-time, for example, `the last ping was 12345 ms ago`. It would be more informative I assume.

@stanislav-grin what do you think?

Reply:
I'm not sure why this is more informative. We may see these logs far later from time this worker killed, or maybe I miss something?

Reply:
do it?

Reply:
for instance, in which log message can you determine the duration more quickly?

```
15:12:55.181 [66633] Worker killed. Worker expired due to heartbeat timeout (10000ms), the last ping was: 2020-10-07T12:12:35.346Z [appId: 1122C599-6A3F-4AEB-8CB3-26EB8CDDF2BA, deployed model: CodelessShoppingCartService, service name: CodelessShoppingCartService, method: getInstructions]
```

or 

```
15:12:55.181 [66633] Worker killed. Worker expired due to heartbeat timeout (10000ms), the last ping was 19835ms ago [appId: 1122C599-6A3F-4AEB-8CB3-26EB8CDDF2BA, deployed model: CodelessShoppingCartService, service name: CodelessShoppingCartService, method: getInstructions]
```

the main reason to show the duration is to check how fast the master process kill the worker, and as we can see on those logs there were ~10 seconds until the worker is killed and this is not good (but it is another issue and we will improve it a little bit later)

Reply:
`00:55:07.281 [70915] Worker killed. Worker expired due to heartbeat timeout (10000ms), the last ping was 16173ms ago [appId: 1122C599-6A3F-4AEB-8CB3-26EB8CDDF2BA, model: default, handler: timer(WWWWWWWWWWWWWWW), file-path: timers/wwwwwwwwwwwwwww.js]`